services/authorization/src/test/kotlin/KeycloakAuthorizationServiceTest.kt

```diff
@@ -1032,4 +1037,26 @@ class KeycloakAuthorizationServiceTest : WordSpec({
             }
         }
     }
+
+    "getUserRoleNames" should {
+        "return all roles of the given user" {
+            val userId = UserId("user-id")
+            val expectedRoleNames = listOf("READER_1", "WRITER_2", "ADMIN_3")
+            val keycloakRoles = expectedRoleNames.mapTo(mutableSetOf()) { name ->
+                Role(
+                    id = RoleId("id_$name"),
+                    name = RoleName(name)
+                )
+            }
+
+            val keycloakClient = mockk<KeycloakClient> {
```

Comment:
Why not use the `KeycloakTestClient` here as well for consistency with the other test cases?